examples/makeSolver.py

```diff
@@ -22,8 +23,13 @@ def main(robot_type, setup_file):
     mpc_model.generateSolver(location=path_to_solvers)
 
 if __name__ == "__main__":
-    if len(sys.argv) < 2:
+    robot_type = "boxer" #options: boxer, po1ntRobot, panda
+    """ Commented out by Saray:
+    if len(sys.argv) < 2 and not input:
         print("Please provide a config file for solver generation.")
+        sys.argv.append('config/' + str(robot_type) + "Mpc.yaml")
     else:
-        robot_type = re.findall('\/(\S*)M', sys.argv[1])[0]
-        main(robot_type, sys.argv[1])
+    robot_type = re.findall('\/(\S*)M', sys.argv[1])[0]
+    """
+    setup_file = 'config/' + str(robot_type) + "Mpc.yaml"
```

Comment:
Why did you remove this functionality of passing the config file through the command line?

Reply:
I changed it back to its original version.